File: backend/src/main/java/com/hrms/dtos/response/PaymentResponseDTO.java
package com.hrms.dtos.response;

import com.hrms.enums.PaymentStatus;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class PaymentResponseDTO {

    private Long paymentId;

    private String message;

    private PaymentStatus status;
    
    private String transactionId;
}

File: payment-service-dotnet/Payment-module/DTOs/PaymentResponseDto.cs
using Payment_module.enums;

namespace Payment_module.DTOs
{
    public class PaymentResponseDto
    {
        public long PaymentId { get; set; }

        public string Message { get; set; }

        public PaymentStatus Status { get; set; }

        public string? TransactionId { get; set; }
    }
}
